tests/unit/test_fee_window.py

```diff
@@ -0,0 +1,263 @@
+    
+    mockUniverse.setIsContainerForReportingParticipant(True)
+    assert mockInitialReporter.callMintFeeTokens(initializedFeeWindow.address, amount) == True
+    assert mockFeeToken.getMintForReportingParticipantTargetValue() == mockInitialReporter.address
+    assert mockFeeToken.getMintForReportingParticipantAmountValue() == amount
+
+@fixture(scope="module")
+def localSnapshot(fixture, augurInitializedWithMocksSnapshot):
+    fixture.resetToSnapshot(augurInitializedWithMocksSnapshot)
+    controller = fixture.contracts['Controller']
+    mockCash = fixture.contracts['MockCash']
+    mockAugur = fixture.contracts['MockAugur']
+    mockFeeToken = fixture.contracts['MockFeeToken']
+    mockFeeTokenFactory = fixture.contracts['MockFeeTokenFactory']
+    mockFeeTokenFactory.setCreateFeeToken(mockFeeToken.address)
+    mockReputationToken = fixture.contracts['MockReputationToken']
+    controller.registerContract(stringToBytes('Cash'), mockCash.address, twentyZeros, thirtyTwoZeros)
+    controller.registerContract(stringToBytes('Augur'), mockAugur.address, twentyZeros, thirtyTwoZeros)
+    controller.registerContract(stringToBytes('FeeTokenFactory'), mockFeeTokenFactory.address, twentyZeros, thirtyTwoZeros)
+    feeWindow = fixture.upload('../source/contracts/reporting/FeeWindow.sol', 'feeWindow')
+    fixture.contracts["initializedFeeWindow"] = feeWindow
+    feeWindow.setController(fixture.contracts["Controller"].address)
+
+    mockUniverse = fixture.contracts['MockUniverse']
+    mockUniverse.setReputationToken(mockReputationToken.address)
+    mockUniverse.setDisputeRoundDurationInSeconds(5040)
+    mockUniverse.setForkingMarket(5040)
+    mockUniverse.setForkingMarket(longToHexString(0))
+    fixture.contracts["Time"].setTimestamp(feeWindowId)
+    feeWindow.initialize(mockUniverse.address, feeWindowId)
+    return fixture.createSnapshot()
+
+@fixture
+def localFixture(fixture, localSnapshot):
+    fixture.resetToSnapshot(localSnapshot)
+    return fixture
+
+@fixture
+def mockUniverse(localFixture):
+    return localFixture.contracts['MockUniverse']
+
+@fixture
+def mockMarket(localFixture):
+    return localFixture.contracts['MockMarket']
+
+@fixture
+def mockCash(localFixture):
+    return localFixture.contracts['MockCash']
+
+@fixture
+def mockReputationToken(localFixture):
+    return localFixture.contracts['MockReputationToken']
+
+@fixture
+def initializedFeeWindow(localFixture):
+    return localFixture.contracts["initializedFeeWindow"]
+
+@fixture
+def constants(localFixture):
+    return localFixture.contracts['Constants']
+
+@fixture
+def mockFeeToken(localFixture):
+    return localFixture.contracts['MockFeeToken']
+
+@fixture
+def mockFeeTokenFactory(localFixture):
+    return localFixture.contracts['MockFeeTokenFactory']
+
+@fixture
+def mockInitialReporter(localFixture):
+    return localFixture.contracts['MockInitialReporter']
+
+@fixture
+def Time(localFixture):
+    return localFixture.contracts["Time"]
```

Comment:
newline at end of file please. You can setup your editor to do this automatically most likely.

Reply:
Thanks, installed 'final-newline' extension into vscode.